MDANSE/Src/MDANSE/Framework/Jobs/VanHoveFunctionDistinct.py

```diff
@@ -411,8 +404,8 @@ def run_step(self, time: int) -> tuple[int, tuple[np.ndarray, np.ndarray]]:
                 frame_index_t1
             )
             coords_t1 = conf_t1["coordinates"][self._indices]
-            direct_cell = conf_t1.unit_cell.transposed_direct
-            inverse_cell = conf_t1.unit_cell.transposed_inverse
+            direct_cell = conf_t1.unit_cell.direct
+            inverse_cell = conf_t1.unit_cell.inverse
 
             scaleconfig_t0 = coords_t0 @ inverse_cell
             scaleconfig_t1 = coords_t1 @ inverse_cell
```

Comment:
I'm a bit confused about this. In UnitCell.py we have 

https://github.com/ISISNeutronMuon/MDANSE/blob/270c6d71d2976ec86057f793dc029a50353bdd5e/MDANSE/Src/MDANSE/MolecularDynamics/UnitCell.py#L52-L79

so the direct matrix is

```math
\begin{bmatrix} 
a_x & a_y & a_z \\
b_x & b_y & b_z \\
c_x & c_y & c_z
\end{bmatrix}
```

where $a_x$, $a_y$, and $a_z$ are the $x$, $y$, and $z$ coordinates of the unit cell vector $a$. If this is true, then the original code with the transpose should be correct. Could it be that during the trajectory conversion, the direct matrix wasn't stored following the above?